Makefile

```diff
@@ -27,15 +27,10 @@ serve:
 	make library
 	LOCAL_MODE=true npx nodemon -x "node -e \"const bootServer = require('./server'); bootServer();\""
 library:
-	cp -r ./lib/ ./server/
-	cp -r ./lib/ ./ui/src/
-	cp -r ./lib/ ./job_runner/
-	cp -r ./lib/ ./oauth_server/
-osxlib: # is this the fix to 'path not found'?
-	cp -r ./lib/ ./server/lib/
-	cp -r ./lib/ ./ui/src/lib/
-	cp -r ./lib/ ./job_runner/lib/
-	cp -r ./lib/ ./oauth_server/lib/
+	rsync -r --exclude="node_modules" --include="*.js"  --include="*/" --exclude="*" ./lib ./ui/src/
+	rsync -r --exclude="node_modules" --include="*.js"  --include="*/" --exclude="*" ./lib ./server/
+	rsync -r --exclude="node_modules" --include="*.js"  --include="*/" --exclude="*" ./lib ./job_runner/
+	rsync -r --exclude="node_modules" --include="*.js"  --include="*/" --exclude="*" ./lib ./oauth_server/
```

Comment:
Sorry, just realized we don't need `./oauth_server` anymore. Can you delete that line (new line 33)? After that, this can merge.

Reply:
Yeah, no problem. I'll take care of it in a sec.